test/built-ins/Proxy/construct/return-not-object-throws-boolean-realm.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
```

Comment:
We have 12 test files like this one: 6 for each primitive type + 6 cross-realm ones.
Do you mind if I make a follow up PR to merge them into 2 files (without loops)?
It feels like they take too much visual space in file system and test run logs.